draft-ietf-lake-traces.md

```diff
@@ -2817,6 +2838,118 @@ OSCORE Master Salt after KeyUpdate (Raw Value) (8 bytes)
 73 ce 79 24 59 40 36 80
 ~~~~~~~~
 
+# Invalid Traces  {#sec-trace-invalid}
+
+This section contains examples of invalid messages, which a compliant implementation must not compose and must or should reject according to {{I-D.ietf-lake-edhoc}}, {{RFC8949}}, {{RFC9053}}, and {{SP-800-56A}}. This is just a small set of examples of different reasons a message might be invalid. The same types of invalidities applies to other fields and messages as well. Implementations should make sure to check for similar types of invalidities in all EHDOC fields and messages.
+
+Invalid encoding of message_1 as array. Correct encoding is a CBOR sequence according to Section 5.2.1 of {{I-D.ietf-lake-edhoc}}.
+
+~~~~~~~~
+Invalid message_1 (38 bytes)
+84 03 02 58 20 74 1a 13 d7 ba 04 8f bb 61 5e 94 38 6a a3 b6 1b ea 5b
+3d 8f 65 f3 26 20 b7 49 be e8 d2 78 ef a9 0e
+~~~~~~~~
+
+Invalid 16-bit encoding 19 00 03 of METHOD = 3. Correct is the deterministic encoding 03 according to Section 1.4 of {{I-D.ietf-lake-edhoc}} and Section 4.2.1 of {{RFC8949}}.
+
```

Comment:
I think it is better to refer to Section 3.1 of -lake-edhoc (instead of Section 1.4).

That's where deterministic encoding is mentioned as a property of the protocol (emphasis mine):

> All **EDHOC messages** are CBOR Sequences [[RFC8742](https://www.rfc-editor.org/info/rfc8742)], and **are deterministically encoded**.

while Section 1.4 is more about terminology and language for the document (emphasis mine):

> When referring to CBOR, this **specification** always **refers** to Deterministically Encoded CBOR as specified in Sections 4.2.1 and 4.2.2 of [[RFC8949](https://www.rfc-editor.org/info/rfc8949)].

Reply:
Done. I will try to update the EDHOC specification during AUTH48 so that section 3.1 refers to 4.2.1 of [[RFC8949](https://www.rfc-editor.org/info/rfc8949)].